jetty-server/src/main/java/org/eclipse/jetty/server/handler/InetAccessHandler.java

```diff
@@ -91,14 +108,55 @@ public void exclude(String pattern)
      */
     public void exclude(String... patterns)
     {
-        _set.exclude(patterns);
+        _addrs.exclude(patterns);
+    }
+
+    /**
+     * Includes a connector name.
+     *
+     * @param name Connector name to include in this handler.
+     */
+    public void includeConnectorName(String name)
```

Comment:
`includeConnectorName` seems a redundant name. I would use `includeConnector(String)`.
Same for the other methods.